src/index.html

```diff
@@ -13,8 +13,24 @@
       />
     </div>
     <div class="links">
-      <div class="link"><a href="slides.html#/">👩‍🏫 Slides</a></div>
-      <div class="link"><a href="labs.html">👩‍🔬 Labs</a></div>
+      <div class="link">
+        <a href="slides.html#/">👩‍🏫 Slides</a>
+        <a
+          href="https://storage.cloud.google.com/zen-formations-pdfs/<%= htmlWebpackPlugin.options.trainingSlug %>/slides.pdf"
```

Comment:
I am afraid of one thing about this link: in development, the PDF content will not be the same as the slides linked just above, WDYT?

Reply:
I agree, the solution proposed in this PR in not good enough.